examples/cifar10/train.cpp

```diff
@@ -26,16 +26,20 @@ ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
 (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
 SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 */
+#include <cstdlib>
 #include <iostream>
 #include <vector>
 #include "tiny_dnn/tiny_dnn.h"
 
+#ifdef _MSC_VER
+#define strcasecmp _stricmp
+#endif
+
 using namespace tiny_dnn;
```

Comment:
No, thats MSVC specific, http://stackoverflow.com/questions/3694723/error-c3861-strcasecmp-identifier-not-found-in-visual-studio-2008